portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp

```diff
@@ -444,41 +444,7 @@ if (sessionError != null) {  %>
 
 <script>
     $(document).ready(function() {
```

Comment:
This file is not being used at all. I'll create another PR just to remote all the ghost files from the old "cross-cancer.do". This file is one of them.